### DISCUSSION TITLE
비글챌 40회차 반성문

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#391] [6. 반성문]
지난 주에 네이버 신입 공채 마감이라 자소서 적느라 정신이 없어 비글챌을 까먹었습니다... 자소서를 계속 고민만하다가 발등에 불이 떨어지고서야 쓰다보니 마음에 안 들어서 시간이 한참 걸렸네요. 그러다 정신을 차려보니 월요일이더라고요 ㅠㅠ 네.. 그래서 못썼습니다.. 

그래도 블로그는 아니지만 노션에 간단하게 네이버 코테 준비로 열심히 알고리즘 문제 푼 거 정리했습니다!  여담으로 어제 네이버 코테쳤는데.. 시간 초과에서 다 걸릴 것 같아서 슬프네요,,

아무튼 앞으로는 열심히 작성하겠습니다!! 다들 화이팅입니다!

**Comment by shkisme** [comment]
바쁘실텐데 화이팅입니다!!! 🔥 네이버 코테 ㄷㄷ! 멋집니다
